lib/parseDateTime.ts

```diff
@@ -1,12 +1,10 @@
-const { DateTime } = require('luxon')
+import { DateTime, Zone } from 'luxon'
```

Comment:
supporting also the `Zone` type is useful within the library? or just an extension point for users?

Reply:
I expected luxon to be hidden as a utility but if it's useful no problem

Reply:
Useful? Maybe
Would it really be used? Probably not...